library/data-model/tests/migrationService.test.ts

```diff
@@ -634,7 +648,7 @@ describe('Migration System Tests', () => {
       );
 
       const migrationDoc = migrationDocs.rows[0].doc as MigrationsDBDocument;
-      expect(migrationDoc.migrationLog.length).toBe(2); // Should still have just the original log entries
+      expect(migrationDoc.migrationLog.length).toBe(3); // Should still have just the original log entries
```

Comment:
Just noting that in the test below this one ('should handle migration failures and update migration log') I get error outputs which is expected because of the test but worrying when watching the tests.  Can we supress output during migrations while testing?

Reply:
Done